_shared/project/Makefile

```diff
@@ -105,15 +104,18 @@ frontend-test:
 {% set target = 'test-py%s'|format(python_version|pyformat(PyFormats.MAJOR_MINOR_FMT)) %}
 .PHONY: {{ target }}
 $(call help,make {{ target }},"run the unit tests in Python {{ python_version|pyformat(PyFormats.MAJOR_DOT_MINOR_FMT) }}")
-coverage: {{ target }}
 {{ target }}: python
 	@pyenv exec tox -qe py{{ python_version|pyformat(PyFormats.MAJOR_MINOR_FMT) }}-tests
 {%- endfor %}
 
 .PHONY: coverage
 $(call help,make coverage,"run the tests and print the coverage report")
 coverage: python
-	@pyenv exec tox -qe coverage
+{%- if cookiecutter._directory == 'pyramid-app' %}
+	@pyenv exec tox -qe 'tests,coverage'
```

Comment:
No point in adding `--parallel` here since we're only running two environments and `coverage` can't start until `tests` finishes